cafebabel/templates/users/macros.html

```diff
@@ -0,0 +1,9 @@
+{% macro social_field(name) %}
+<li>
+  <a href=# class=icon-{{ name }}></a>
+  <label for=social-{{ name }}>
+    <span class=icon-{{ name }}></span>
+    <input type=url id=social-{{ name }} name=socials-{{ name }} value="{{ user.profile.socials.get(name) }}">
+  </label>
+</li>
+{% endmacro %}
```

Comment:
same : why is it called `macros` ? For me it doesn't say enough what it is about

Reply:
I was thinking of storing all macros that we may have for the users module. Just like we have users/views for the views and users/models for the models, each having the users related functionalities.
For now we only have the `social_field` macro though.
How would you see this?

Reply:
ok makes more sense as long as there are severals. But still I would pick a name that shows what macro it is like `profile-marcos`?

Reply:
Don't you feel like it would be redundant to have `users/profile-macros.html`?
When we'll have some for the article, does it mean we'd call them `articles/articles-macros.html`?

Reply:
we'll have `input id=social-{{ name }}..` only on profile since we can register the social only here. That's why i suggested `profile-macros`

Reply:
indeed, that's why I put them in the `users/` folder ;)
Anyways, I'll call it _users/profile-macros.html_ if it makes more sense to you 🤝

Reply:
cool !